src/Frontend/components/portal/CreateAccount.js

```diff
@@ -10,24 +10,71 @@
 
 'use strict';
 
-import React     from 'react';
-import Translate from 'react-translate-component';
-import Form      from './signup/Form';
-import InfoBox   from './signup/InfoBox';
+import React, { Component, PropTypes }     from 'react';
+import Translate                           from 'react-translate-component';
+import { connect            }              from 'react-redux';
+import { bindActionCreators }              from 'redux';
+import Form                                from './signup/Form';
+import InfoBox                             from './signup/InfoBox';
+import * as userActions                    from '../../actions/userActions';
 
 /**
- * Markup component which renders the `create account` page.
+ * Presentational component for the sign-up page
  *
- * @returns {React.Element} The markup.
+ * @author Benjamin Bieler <ben@benbieler.com>
  */
-export default () => {
-  return (
-    <div>
-      <h1><Translate content="pages.portal.head" /></h1>
+class CreateAccount extends Component {
+
+  static PropTypes = {
```

Comment:
the `propTypes` property starts with a lowercase char, otherwise React doesn't care about this prop-type definition